core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -308,14 +308,14 @@ oppia.directive('conversationSkin', [
             return PlayerCorrectnessFeedbackEnabledService.isEnabled();
           };
 
-          $scope.isOnLastCard = function () {
-            return PlayerTranscriptService.isLastCard(
-              PlayerPositionService.getActiveCardIndex());
+          $scope.isInNavigation = function () {
+            return PlayerPositionService.getInNavigation();
           };
 
           $scope.isCorrectnessFooterEnabled = function() {
             return (
-              $scope.answerIsCorrect && $scope.isCorrectnessFeedbackEnabled());
+              $scope.answerIsCorrect && $scope.isCorrectnessFeedbackEnabled() &&
+              !$scope.isInNavigation());
```

Comment:
I think this "is in navigation" idea is a bit hard to pin down clearly. If I'm reading an older card it seems a bit strange to say I'm "in navigation".

I think the concept you're trying to express is something along the lines of learnerJustSubmittedAnAnswer. Can we use that instead? Set that to true when an answer is submitted, and to false once any navigation button is pressed.